Cabal/ChangeLog.md

```diff
@@ -1,3 +1,6 @@
+# 3.12.0.0 [Hécate](mailto:hecate+github@glitchbra.in) March 2024
```

Comment:
@ffaf1: if that's not a problem, would you mind setting your own email address here (and elsewhere), since you are performing (most of) the work for this lightning release?